rock_paper_scissors.py

```diff
@@ -97,11 +97,14 @@ def rounds(self):
 
     @rounds.setter
     def rounds(self, value):
-        if value % 2 != 0:
-            self._rounds = value
-        else:
+        try:
+            value = int(value)
+            if value % 2 == 0 or value < 0:
+                raise ValueError
```

Comment:
In this instance, I think what you have works.  However, I want to point out that generally it's a good idea to reserve try/except blocks for as few potential error conditions as possible so that you have the best idea of what the original cause of the error was.

For example, if we wanted to be more specific with the error message, we'd want to break this out into more discrete cases:
```python
try:
    value = int(value)
except ValueError:
    print('Input cannot be a string')

try:
    assert value % 2 == 0
except AssertionError:
    print('Please enter an odd number of rounds')

try:
    assert value > 0
except AssertionError:
    print('Please enter a number greater than zero')
```
I think what you have here is solid, though, because the same error recovery process and the error message is the same in all three possible error scenarios*, and it's also arguably appropriate to raise a `ValueError` in these cases.

*Technically negative numbers can still be odd 🤓